LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Rishi
+Copyright (c) 2015-2016 Rishi
```

Comment:
Lets just get rid of the year altogether and just keep it:
Copyright (c) Rishi Jain